scripts/docstrings_checker_test.py

```diff
@@ -21,6 +21,7 @@
 
 
 class ASTDocStringCheckerTest(unittest.TestCase):
```

Comment:
Does this need to be ASTDocstringCheckerTest? As Docstring is a single word right?

Reply:
Done